project/SotaBuild.scala

```diff
@@ -89,6 +91,18 @@ object SotaBuild extends Build {
       testOptions in RandomTests += Tests.Argument(TestFrameworks.ScalaTest, "-n", "RandomTest"),
       parallelExecution in Test := true,
       dockerExposedPorts := Seq(8081),
+      dockerCommands := dockerCommands.value.filterNot {
```

Comment:
Hmm...I don't know, and couldn't find in the documentation, how to add an arbitrary file from sbt-native-packager. What we could do, however, is update the advancedtelematic/java:openjdk-8-jre image (the base image for these builds) to include the script. If we did that, actually, we could get away with making no changes to the build process at all, and pass the wait-for-it entrypoint in the docker-compose yaml file. That would maybe be the purest way to go...